python-lib/prologin/concours/stechec/models.py

```diff
@@ -22,6 +22,17 @@ def strip_ansi_codes(t):
     return stripper_re.sub('', t)
 
 
+def bulk_create(model, objs):
```

Comment:
Maybe change the name to avoid confusion? bulk_create_return_ids?

Reply:
Done.